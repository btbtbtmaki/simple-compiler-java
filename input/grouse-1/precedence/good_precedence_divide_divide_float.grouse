main {

  print 6.3 / 8.6 / 7.3 nl;
  print 2.6 / 8.5 / 6.6 nl;
  print 3.6 / 3.6 / 4.9 nl;
  print 3.8 / 9.8 / 8.7 nl;
  print 9.4 / 1.6 / 6.4 nl;
  print 7.5 / 3.9 / 5.4 nl;
  print 6.3 / 8.7 / 4.8 nl;
  print 8.4 / 9.8 / 5.4 nl;
  print 5.4 / 3.9 / 9.5 nl;
  print 9.4 / 6.0 / 6.3 nl;
  print 4.2 / 7.1 / 5.8 nl;
  print 5.8 / 8.1 / 6.5 nl;

  print 5.4 / 2.1 / 7.0 nl;
  print 6.2 / 3.9 / 3.8 nl;
  print 3.8 / 9.7 / 1.2 nl;
  print 4.3 / 2.8 / 5.4 nl;
  print 2.6 / 8.3 / 5.3 nl;
  print 1.5 / 4.3 / 8.6 nl;
  print 4.5 / 2.7 / 5.4 nl;
  print 7.3 / 1.4 / 1.4 nl;
  print 5.4 / 4.5 / 7.0 nl;
  print 3.6 / 2.9 / 2.4 nl;
  print 5.1 / 8.2 / 4.2 nl;
  print 6.7 / 4.8 / 3.9 nl;

  print 8.9 / 3.5 / 9.2 / 7.9 nl;
  print 8.5 / 7.6 / 6.5 / 5.2 nl;
  print 4.8 / 2.8 / 1.1 / 8.7 nl;
  print 1.1 / 8.9 / 4.9 / 8.1 nl;
  print 1.5 / 5.4 / 7.7 / 8.3 nl;
  print 7.6 / 8.8 / 3.8 / 7.6 nl;
  print 3.1 / 9.6 / 6.5 / 8.8 nl;
  print 9.4 / 3.9 / 6.7 / 2.2 nl;
  print 4.6 / 5.2 / 3.5 / 3.9 nl;
  print 6.2 / 9.8 / 4.4 / 1.0 nl;
  print 5.4 / 4.6 / 7.1 / 2.2 nl;
  print 8.2 / 8.4 / 8.3 / 7.4 nl;

  print 7.0 / 4.1 / 2.6 / 7.1 nl;
  print 6.3 / 2.4 / 5.7 / 9.0 nl;
  print 1.0 / 1.3 / 4.7 / 3.1 nl;
  print 4.5 / 4.9 / 8.6 / 1.9 nl;
  print 4.6 / 4.7 / 2.7 / 5.2 nl;
  print 7.7 / 8.8 / 1.9 / 4.6 nl;
  print 2.3 / 6.8 / 3.9 / 8.9 nl;
  print 2.3 / 6.2 / 2.6 / 9.6 nl;
  print 3.2 / 2.9 / 8.3 / 5.3 nl;
  print 9.0 / 5.0 / 5.6 / 4.3 nl;
  print 7.5 / 5.4 / 7.3 / 2.8 nl;
  print 7.4 / 4.4 / 3.9 / 4.7 nl;

}

