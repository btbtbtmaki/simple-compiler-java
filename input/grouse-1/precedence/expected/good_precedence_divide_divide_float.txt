0.10035
0.0463458
0.204082
0.0445696
0.917969
0.356125
0.150862
0.15873
0.145749
0.248677
0.101991
0.110161
0.367347
0.418354
0.32646
0.284392
0.0591043
0.0405625
0.308642
3.72449
0.171429
0.517241
0.148084
0.357906
0.034987
0.0330894
0.179131
0.00311402
0.00434639
0.0299043
0.0056454
0.163518
0.064807
0.143785
0.0751545
0.0158937
0.0924874
0.0511696
0.0527955
0.0562036
0.0697096
0.100114
0.00974461
0.0148625
0.0250841
0.0747508
0.0679496
0.0917522
